gitlab-ci/README.md

```diff
@@ -25,6 +25,58 @@
 - GitLab CI script:
     - `.gitlab-ci.yml`
 
+## GitLab runner configuration
+
+Some of the jobs in the CI script require more CPU or memory than available
+to GitLab shared runners.  These jobs are tagged to run on project-specific
+runners hosted by VMware, e.g.:
+
+```
+.test-imported-souffle
+    extends: .install-ddlog
+    tags:
+        - ddlog-ci-1
+```
+
+These runners are attached to the DDlog GitLab organization, so in order to use
+them with your fork of the DDlog repo, you must transfer the fork (or the GitLab
+mirror of the fork) to the DDlog organization (see below).
+
+I followed these [instructions](https://docs.gitlab.com/runner/install/linux-manually.html)
+to configure the GitLab runners.
+
+A copy of `/etc/gitlab-runner/config.toml`:
+
+```
+concurrent = 8
+check_interval = 0
+
+[session_server]
+  session_timeout = 1800
+
+[[runners]]
+  name = "ddlog-ci-1"
+  url = "https://gitlab.com"
+  token = "Bjx-aJUGEH6Fp6rBEXkf"
```

Comment:
is it ok to have this here in clear?

Reply:
As far as I can tell, anyone with this token can register their worker to run our tests. They're welcome to do so :) The worst a malicious worker should be able to do is report success when the test should actually fail.  But at least we will see in GitLab that external workers are being used and blacklist them if necessary.